samples/matter/lock/Kconfig

```diff
@@ -77,6 +77,16 @@ config CHIP_ENABLE_SLEEPY_END_DEVICE_SUPPORT
 
 endif # NET_L2_OPENTHREAD
 
+if CHIP_WIFI
+
+# Enable Legacy Power Save mode by default.
+# The device sleeps most of the time and wakes up on each DTIM interval.
+config NRF_WIFI_LOW_POWER
```

Comment:
We have this set to 'n' in Kconfig.defaults in the sdk-connectedhomeip. Should we turn this feature on overall for Matter there? If we want to use it only for lock sample, maybe it's better to explicitly enable it in the prj.conf anyway.